packages/web/src/pages/Honeypots/VaultDetailsPage/Sections/VaultSubmissionsSection/VaultSubmissionsSection.tsx

```
@@ -38,13 +42,36 @@ export const VaultSubmissionsSection = ({ vault }: VaultSubmissionsSectionProps)
+    if (savedSubmissions?.length === 0) return null;
+
+    return (
+      <div className="bonus-points-section">
+        <div className="title-container">
+          <div className="icon">
+            <VerifiedIcon />
+          </div>
+          <p>{t("bonusPointsTitle")}</p>
+        </div>
+
+        <div className="content" dangerouslySetInnerHTML={{ __html: t("bonusPointsDescription") }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)